superset-frontend/src/dashboard/components/gridComponents/ChartHolder.jsx

```diff
@@ -183,6 +200,29 @@ class ChartHolder extends React.Component {
         ? parentComponent.meta.width || GRID_MIN_COLUMN_COUNT
         : component.meta.width || GRID_MIN_COLUMN_COUNT;
 
+    let fullStyle = {};
+    let w = 0;
+    let h = 0;
```

Comment:
prefer full word variable names, so `height` and `width` or `chartHeight` and `chartWidth`

Reply:
Changed c0526bd607a84ebc9e32f7d4676af14a06a2624b